src/index.css

```diff
@@ -1,10 +1,34 @@
 @font-face {
+  src: url('./fonts/BCSans-Regular.woff2') format('woff2'),
+    url('./fonts/BCSans-Regular.woff') format('woff');
+  font-weight: 400;
+  font-style: normal;
+  font-family: 'BC Sans';
+}
+@font-face {
+  src: url('./fonts/BCSans-BoldItalic.woff2') format('woff2'),
+    url('./fonts/BCSans-BoldItalic.woff') format('woff');
+  font-weight: 700;
+  font-style: italic;
```

Comment:
did you mean for the font style to be 'oblique' since it's bold & italic?

Reply:
I had to look it up what `oblique` does. Looks like it does a very similar effect to `italic`! so I would like to keep as they are now.